AssetRipperLibrary/Ripper.cs

```diff
@@ -224,8 +224,9 @@ private void OverrideNormalExporters()
 			OverrideExporter<ISprite>(textureExporter);
 
 			//Shader exporters
-			OverrideExporter<IShader>(new DummyShaderTextExporter());
 			OverrideExporter<IShader>(new SimpleShaderExporter());
+			OverrideExporter<IShader>(new DummyShaderTextExporter(Settings));
+			OverrideExporter<IShader>(new YAMLShaderExporter(Settings));

```

Comment:
`SimpleShaderExporter` should come after the other two so that it's on top of the stack.